src/acl/ACL.ts

```diff
@@ -147,7 +155,7 @@ class ACL {
         Record<NodeId, Permission>
       > = {};
       for await (const o of this.aclNodesDb.createReadStream()) {
-        const nodeId = (o as any).key as NodeId;
+        const nodeId = IdInternal.fromBuffer<NodeId>((o as any).key)!;
```

Comment:
@tegefaulkes I believe you can shorten this to just `Id.create<NodeId>((o as any).key)`.

The `!` here asserts that it works. But if you know it works, you can also just pass the buffer directly into `create` method, rather than `fromBuffer`.